src/btreemap.rs

```diff
@@ -570,6 +570,38 @@ where
             .map(V::from_bytes)
     }
 
+    /// Removes and returns the last element in the map. The key of this element is the maximum key that was in the map
+    pub fn pop_last(&mut self) -> Option<(K, V)> {
+        if self.root_addr == NULL {
+            return None;
+        }
+
+        let root_node = self.load_node(self.root_addr);
+        root_node.max_key(self.memory()).map(|max_key| {
```

Comment:
Why don't we use `root_node.get_max` here?

Reply:
get_max and remove call value reading twice, which is unnecessary. Especially when there are N items of data, this overhead will be * N

Reply:
Reading values are cheap, so I would keep the code simple by reusing the same code and only optimize it if a benchmark indicates that it would lead to a significant performance boost.